webpages/js/tools.js

```diff
@@ -319,6 +319,10 @@
     _.fillEls('.fnOrYour', y ? 'Your' : n + "'s");
     _.fillEls('.fnOryou',  y ? 'you'  : n       );
 
+    // Used on profile page, prefixes 'Your ' if you're signed in and the page
+    // is about you. The space is needed for formatting.
+    _.fillEls('.yourOrBlank', y ? 'Your ' : '' );
```

Comment:
this should be `.YourOrBlank` because the case of "you/Your" is consistent between the class and the string we put in

Nicely done with the space in the string.